packages/shared/src/components/squads/SquadHeaderBar.tsx

```diff
@@ -92,18 +92,19 @@ export function SquadHeaderBar({
         className,
       )}
     >
+      <SquadMemberShortList
+        squad={squad}
+        members={members}
+        size={ProfileImageSize.Small}
+      />
       {showJoinButton && (
         <SquadJoinButton
-          squad={squad}
+          joinText="Join"
           origin={Origin.SquadPage}
           size={ButtonSize.Small}
+          squad={squad}
         />
       )}
-      <SquadMemberShortList
-        squad={squad}
-        members={members}
-        size={ProfileImageSize.Small}
-      />
```

Comment:
Based on this [thread](https://dailydotdev.slack.com/archives/C069XNYMDGV/p1725894965874349?thread_ts=1725887388.352119&cid=C069XNYMDGV), let's revert these changes.

Reply:
Removed! ✅

Reply:
@ilasw also the change on this file should have been reverted